src/gui/folderman.cpp

```diff
@@ -786,8 +786,8 @@ QString FolderMan::checkPathValidityForNewFolder(const QString &path) const
 
 QString FolderMan::findGoodPathForNewSyncFolder(const QString &basePath, const QString &newFolder) const
 {
-    // reserve 3 characters to allow appending of a number 0-100
-    const QString normalisedPath = FileSystem::createPortableFileName(basePath, FileSystem::pathEscape(newFolder), 3);
+    // reserve 3 characters to allow appending of a number 0-100, plus 2 for " (" and 1 for ")"
```

Comment:
we could use `std::stringview(" (100)").size()` to have a compile time placeholder?

Reply:
done